app/models/user.rb

```diff
@@ -218,10 +218,9 @@ def student_team
 
   private
 
-  # Ensures that the location column either contains non-whitespace text, or is NULL
-  # This ensures that sorting by location yields useful results
-  def sanitize_location
-    self.location = nil if self.location.blank?
+  def format_location
+    self.location = nil if self.location.blank? # for sorting
+    self.location = self.location.downcase.titlecase unless location.nil?
```

Comment:
We could also add a `String#strip`. The entire method body could be condensed to:

```ruby
self.location = location.to_s.strip.titlecase.presence
```